spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context 'date_to_string' do
+    it 'formats a date' do
+      expect(date_to_string(Date.current)).to be_a_kind_of String
```

Comment:
Is there value in testing that the method not merely returns a string but a correctly formatted date? i.e.
```ruby
expect(date_to_string(Date.current)).to eq Date.current.strftime('%d %B %Y')
```
It's very redundant, but the current test doesn't seem to give much.